terraform/modules/region/vpc.tf

```diff
@@ -1,3 +1,9 @@
-resource "aws_security_group" "lambda_egress" {
-  name = "lpa-uid-${local.environment_name}"
+resource "aws_security_group" "lambda" {
+  name_prefix = "lpa-uid-${local.environment_name}"
```

Comment:
I have a feeling `name_prefix` is preferred now, but I may well have got that wrong.

Reply:
I don't actually like name prefix as it makes data sources on a real pain.
But it's up to you 😃